pkg/action/hooks.go

```diff
@@ -107,18 +103,6 @@ func (cfg *Configuration) execHook(rl *release.Release, hook release.HookEvent,
 	return nil
 }
 
-// hookByWeight is a sorter for hooks
-type hookByWeight []*release.Hook
-
-func (x hookByWeight) Len() int      { return len(x) }
-func (x hookByWeight) Swap(i, j int) { x[i], x[j] = x[j], x[i] }
```

Comment:
did we understood why they used to sort first by kind and then by name in a separate package?

Reply:
I did never. I mentioned it in the upstream PR, expecting maintainers to push back if it was intentional.